lms/envs/common.py

```diff
@@ -1915,7 +1915,7 @@
     'edxnotes',
 
     # Splash screen
-    'splash',
+    # 'splash',
```

Comment:
This is also causing `'Cannot add foreign key constraint'` error. The repo is still on Django 1.4 and inherited from ConfigurationModel, changed_by field in that model causing the mentioned error.